src/utils/image_server.cljs

```diff
@@ -265,6 +265,10 @@
                             :ring-width               ring-width})))
 
 (defn get-account-qr-image-uri
+  {:deprecated
+   "The size of the avatar image inside this QR is not the same as designs and it's not
+   returning the expected image.
+   Please use `quo2.components.share.qr-code.view/view` in its `:profile` variant."}
```

Comment:
Function now deprecated, also it had a previous use which has been removed so no warnings from this function are thrown

Reply:
@siddarthkay  The deprecation has been removed